upgrade/v18xto190/upgrade.go

```diff
@@ -122,31 +122,55 @@ func upgradeSettings(namespace string, lhClient *lhclientset.Clientset, resource
 }
 
 func updateOrphanResourceAutoDeletionSetting(settingsMap map[string]*longhorn.Setting) (err error) {
-	oldSetting, exist := settingsMap[string(types.SettingNameOrphanAutoDeletion)]
+	oldOrphanReplicaDataAutoDeletionSetting, exist := settingsMap[string(types.SettingNameOrphanAutoDeletion)]
```

Comment:
The naming is a bit confusiung, like we have this setting with the previous version.